dime-coding-standards/checklists/tex/peer_code_review_guidance_note.tex

```diff
@@ -0,0 +1,127 @@
+
+\section*{How it works}
+
+	\begin{enumerate}
+	    
+	  
+		\setlength\itemsep{-0.1em}
+		\item Teams \href{https://survey.wb.surveycto.com/collect/code_review_sign_up?caseid=}{sign-up} to participate in the code review each quarter. All DIME Research Assistants working on code are expected to participate, and any Bank staff or consultant who would like to get feedback on their code is welcome to participate.
+    
+
+		\item All participants prepare a code package to submit to review, following \href{https://github.com/worldbank/dime-standards/blob/master/dime-coding-standards/checklists/Peer%20Code%20Review%20Submission%20Checklist.pdf}{these guidelines}. Submitted code should be \textbf{no longer than 1000 lines}. 
+        
+        \begin{itemize}
+            \item As part of this step, teams should evaluate if they would like to receive comments on \textbf{code reproducibility} and \textbf{output replicability}. 
+            \item \textbf{Note} that in this case, teams should prepare a \textbf{de-identified} version of the dataset to include in the submission package.
+        \end{itemize}
+
+        \item Participants are then paired for review based on preferred statistical software and length of code.
+
+        \item The actual peer code review activity takes place over the course of one week, which includes:
+         \begin{itemize}
+            \item A \textbf{kickoff session} with an overview of the process, and pair assignments. 
+            \item A \textbf{group work party} with technical support from DIME Analytics. Pairs work together, and answer questions to allow each other to review their submitted code.
+        \end{itemize}
+
+        \item Reviewers finally submit feedback using \href{https://survey.wb.surveycto.com/collect/code_review_summary?caseid=}{this form}. They also fill out and submit the following checklists:
+        \begin{itemize}
+            \item \textbf{Mandatory}: \href{https://github.com/worldbank/dime-standards/blob/master/dime-coding-standards/checklists/Reviewer%20Feedback%20Checklist.pdf}{Reviewer Feedback Checklist}
+            \item \textbf{Depending on tasks in submitted code:} 
+            \begin{itemize}
+                \item \href{https://github.com/worldbank/dime-standards/blob/master/dime-coding-standards/checklists/Cleaning%20Code%20Review%20Checklist.pdf}{Cleaning Checklist}
+                \item \href{https://github.com/worldbank/dime-standards/blob/master/dime-coding-standards/checklists/Construction%20Code%20Review%20Checklist.pdf}{Construction Checklist}
+                \item \href{https://github.com/worldbank/dime-standards/blob/master/dime-coding-standards/checklists/Analysis%20Code%20Review%20Checklist.pdf}{Analysis Checklist}
+            \end{itemize}
```

Comment:
INSERT:          
\item At the end of the week, reviewers submit \href{https://survey.wb.surveycto.com/collect/code_review_summary?caseid=}{a structured feedback form} and upload their completed checklists.